cmd/restic/cmd_check.go

```diff
@@ -42,10 +46,27 @@ func init() {
 
 	f := cmdCheck.Flags()
 	f.BoolVar(&checkOptions.ReadData, "read-data", false, "read all data blobs")
+	f.UintSliceVar(&checkOptions.ReadDataSubset, "read-data-subset", nil, "read subset of data packs")
 	f.BoolVar(&checkOptions.CheckUnused, "check-unused", false, "find unused blobs")
 	f.BoolVar(&checkOptions.WithCache, "with-cache", false, "use the cache")
 }
 
+func checkFlags(opts CheckOptions) error {
+	if opts.ReadData && opts.ReadDataSubset != nil {
+		return errors.Errorf("check flags --readData and --read-data-subset cannot be used together")
+	}
+	if opts.ReadDataSubset != nil {
+		if len(opts.ReadDataSubset) != 2 {
+			return errors.Errorf("check flag --read-data-subset must have two values")
```

Comment:
Use `errors.Fatalf()` here, so that no stack trace is printed (we need to rework error handling in restic, badly). Also, maybe add an example?